src/gui/DatabaseWidgetStateSync.cpp

```diff
@@ -48,8 +50,11 @@ void DatabaseWidgetStateSync::setActive(DatabaseWidget* dbWidget)
     if (m_activeDbWidget) {
         m_blockUpdates = true;
 
-        if (!m_splitterSizes.isEmpty())
-            m_activeDbWidget->setSplitterSizes(m_splitterSizes);
+        if (!m_mainSplitterSizes.isEmpty())
```

Comment:
Last minute edit, can you insert bracket here and to the below if? Thanks

Reply:
ok